lib/rubocop/cop/rails/bulk_change_table.rb

```diff
@@ -113,8 +113,10 @@ class BulkChangeTable < Base
         MYSQL_COMBINABLE_ALTER_METHODS = %i[rename_column add_index remove_index].freeze
 
         POSTGRESQL_COMBINABLE_TRANSFORMATIONS = %i[change_default].freeze
+        POSTGRESQL_COMBINABLE_TRANSFORMATIONS_6_1_PLUS = %i[change_null].freeze
```

Comment:
Can you update the name to something like the following?

```suggestion
        POSTGRESQL_COMBINABLE_TRANSFORMATIONS_SINCE_6_1 = %i[change_null].freeze
```